src/main/java/uk/gov/ons/ctp/response/casesvc/CaseSvcApplication.java

```diff
@@ -247,4 +255,17 @@ public DistributedLockManager reportDistributedLockManager(RedissonClient rediss
   public Clock clock() {
     return Clock.systemDefaultZone();
   }
+
+  private static final String COLLEX_CACHE_NAME = "collectionexercises";
+
+  @Bean
+  public CacheManager cacheManager() {
+    return new ConcurrentMapCacheManager(COLLEX_CACHE_NAME);
+  }
+
+  @CacheEvict(
+      allEntries = true,
+      cacheNames = {COLLEX_CACHE_NAME})
+  @Scheduled(fixedDelay = 60000)
+  public void cacheEvict() {}
```

Comment:
Maybe put a comment in the method to say "this is getting rid of cache entries, in case anything's been changed - we imagine that the maximum of 1 minute delay to seeing changes reflected in the case service will not cause any issues"

Reply:
@NickGrantONS Done